src/components/Input/input.scss

```diff
@@ -43,3 +47,7 @@ input::placeholder {
     }
   }
 }
+.error-message {
+  color: $error;
+  font-size: 16px;
```

Comment:
rem calculation: 16/18 = 0.89rem

Reply:
I changed it to rems, only it seems base is 16, so i changed error font to 1rem, and wrapper to 5rem.